### DISCUSSION TITLE
Welcome to WiFi-remote-for-Bestway-Lay-Z-SPA Discussions!

### DISCUSSION THREAD

**Discussion by visualapproach** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#43] [General]
## 👋 Welcome!
  Trying Discussions as a place to chat, instead of Issues
  You can still open issues.

**Comment by Inazone** [comment]
Hi there,
I am very interested in your project. However I am a total noob to programming, although I am handy with a soldiering iron. Do you know whether your project would work with a Maldives pump controller? I cannot believe lazy spa hasn't done this kind of work already.... The 72hr timer thing is driving me insane... Hence my search for a solution, despite the really deep water I feel I'm in with looking at projects like this.
Keep up the great work. 

Many thanks 
Alex.

**Reply by visualapproach** [reply]
Hi, and thank you @Inazone ! Well, actually they have done a wifi pump. For £299.99!! Which is about £290 more than this 😬 .
I have no idea if your pump is compatible with this project, but chances are it is. If the display is similar and you have a 6-wire ribbon cable between the display and the pump. The 72-hour annoyance will be gone if you build this, but the water pump is probably going to fail after 6 month continuous use. At least my first and second pump did... 
Have fun!

**Reply by mhl66** [reply]
What a great project! Congratulations! News from Bestway: The new Maldives and Hawaii Lay-Z-Spa-HydroJet Pro models do have an additional outlet for the HydroJet function, driven by a separate pump. I am a proud owner of the Maldives model, just as @Inazone - Any chance to support the HydroJet pump?

**Comment by cfarrar1** [comment]
Hi,
I am looking to do this upgrade to my spa however I have to buy a new heater/control unit first.
Unfortunately the sellers don’t list if the cable is 6 or 4 wire version. 🙄

What version is everyone using? Ie 54183?

Thanks!

**Comment by wyliecoyoteuk** [comment]
easiest way to find out is to take the top off the pump

**Comment by wyliecoyoteuk** [comment]
Sorry, misread.
Depends on which version you buy.
The new pumps are quite different, If you are buying a second hand one,
check it out when you get it and decide then

**Comment by iistelladx** [comment]
hi, i think that i have the same pump but i never make nothing with arduino or somethings like that. if i check if my pump has the 6 or 4 wire version can you make it for me and send it to me? :D :D obviously i'll pay you!

**Comment by hjkoster400d** [comment]
Helsinki 2021 model is working. Pump model S100101. User 680ohm resistors between LLC and disp, flashed model MIAMI2021 AMD i'm using PCB V1. Thanks for the awesome project. Gr Rik
